pages/pipelines/hosted_agents/overview.md

```diff
@@ -2,19 +2,20 @@
 
 Buildkite hosted agents provides a fully-managed platform on which you can run your agents.
```

Comment:
Hi @LiamPClancy ,

Would you be OK with this suggested change?

I think the main issue @mbelton-buildkite had with the original wording was that this statement didn't point out the contrast with self-hosted agents (i.e. to make Buildkite hosted agents a distinguishing feature).

```suggestion
Buildkite hosted agents provides a fully-managed platform on which you can run your agents, so that you don't have to manage agents in your own self-hosted environment.
```

Reply:
Darn, I forgot to commit this before merging, so I will make this part of the clusters GA release doc updates.

Reply:
Unfortunately I committed this after merging this PR 🤦 , and so I'm implementing this change as part of #2642 .